packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/KycVerification/ExtraFields/template.success.tsx

```diff
@@ -213,6 +213,23 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
     }
   }
 
+  const updateMultiSelectItem = (nodeId: string, allSelectedItems: Array<string>) => {
+    props.formActions.change(KYC_EXTRA_QUESTIONS_FORM, nodeId, nodeId)
+
+    const { blocking, context, nodes } = props.extraSteps
+
+    nodes.map(
```

Comment:
It is a bit confusing to see `map` being used solely to mutate the inner values. I would rather change the mapper function appropriately and assign the result of the `map` back to `props.extraSteps.nodes` (preferable) or use `forEach` instead.